src/NovelRT.Interop/CMakeLists.txt

```diff
@@ -43,7 +43,11 @@ set(SOURCES
 add_library(Interop SHARED ${SOURCES})
 add_dependencies(Interop Engine)
 
-set_property(TARGET Interop PROPERTY OUTPUT_NAME "NovelRT.Interop-$<CONFIG>")
+if(CMAKE_BUILD_TYPE MATCHES RELEASE)
```

Comment:
Use a generator expression for this - it won't work for multi-config generators.

Something like `$<IF:<CONFIG:Release>,,-$<CONFIG>>` should work.

Reply:
Good catch, ty